README.md

```diff
@@ -72,7 +72,7 @@ $ python setup.py install
 
 ## Documentation
 
-Documentation is available online: https://milvus.io/api-reference/pymilvus/v2.2.3/About.md
+Documentation is available online: https://milvus.io/api-reference/pymilvus/v2.2.4/About.md
```

Comment:
The valid document link is `v2.2.6` now, please help to change it

Reply:
fixed.